galaxy.yml

```
@@ -1,7 +1,7 @@
 ---
 namespace: infra
 name: controller_configuration
-version: 2.7.1-devel
+version: 2.9.0-devel-3
```

Happy for this to be updated, but could it please be left as `2.9.0-devel` please?

```suggestion
version: 2.9.0-devel
```